README.rst

```diff
@@ -99,6 +99,21 @@ More information on compilation and development can be found on the
 of the project documentation.
 
 
+Extra
+=====
+
+Make sure Avahi is installed if you are using Linux. Avahi provides network
+discovery for NDI devices.
+
+Ubuntu
+^^^^^^
+
+.. code-block::
+
+    sudo apt install avahi-daemon
```

Comment:
Shouldn't this be `apt-get`?

Reply:
For the most part they are interchangeable, with `apt` being intended for end users (see [apt man](https://manpages.ubuntu.com/manpages/oracular/en/man8/apt.8.html)). Perhaps user preference (or less keys to press 😉).

```
apt provides a high-level commandline interface for the package management system. It is
intended as an end user interface and enables some options better suited for interactive
usage by default compared to more specialized APT tools like [apt-get](https://manpages.ubuntu.com/manpages/oracular/en/man8/apt-get.8.html)(8) and [apt-cache](https://manpages.ubuntu.com/manpages/oracular/en/man8/apt-cache.8.html)(8).
```